ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -478,12 +478,12 @@ static KsqlRestApplication buildApplication(
         restConfig.getCommandConsumerProperties(),
         restConfig.getCommandProducerProperties());
 
-    final StatementExecutor statementExecutor =
-        new StatementExecutor(serviceContext, ksqlEngine, hybridQueryIdGenerator);
+    final InteractiveStatementExecutor interactiveStatementExecutor =
```

Comment:
no need to change the variable name.  keep the shorter name IMHO.

Reply:
Ack